org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/SimpleGUI.java

```diff
@@ -186,6 +188,9 @@
  * (1) the run() method in SatRunner is launched from a fresh thread <br>
  * (2) the run() method in the instance watcher (in constructor) is launched
  * from a fresh thread
+ *
+ * @modified: Nuno Macedo, Eduardo Pessoa // [HASLab] electrum-base,
```

Comment:
small changes to support new solving and iteration modes; also, new temporal examples added;